dmscripts/insert_framework_results.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import unicode_literals
+import unicodecsv as csv
```

Comment:
Also, this package isn't in the requirements.  Newest version is `unicodecsv==0.14.1`.